### PR TITLE
Fixed Mac bindings

### PR DESCRIPTION
**Pull request by herrjemand**
withoutResponse can be optional

### CODE REVIEW

File: lib/mac/src/noble_mac.mm [hunk 1]
```diff
@@ -141,12 +141,18 @@
 // write(deviceUuid, serviceUuid, characteristicUuid, data, withoutResponse)
 Napi::Value NobleMac::Write(const Napi::CallbackInfo& info) {
     CHECK_MANAGER()
-    ARG5(String, String, String, Buffer, Boolean)
```
Comment (rzr) [inline lib/mac/src/noble_mac.mm:144]: isnt it preferable to add comments like :

    ...  /* , Boolean */)

**Review by rzr — COMMENTED** [review]
feedback on comment welcome

File: lib/mac/src/noble_mac.mm (hunk 1 above)
Comment (herrjemand) [reply lib/mac/src/noble_mac.mm:144]: @rzr Yeaj, that's a good idea

**Comment by herrjemand** [comment]
@rzr done!

**Review by rzr — APPROVED** [review]

**Pull request merged**